airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -129,7 +129,7 @@ def _render_log_id(self, ti: TaskInstance, try_number: int) -> str:
         return self.log_id_template.format(
             dag_id=ti.dag_id,
             task_id=ti.task_id,
-            run_id=ti.run_id,
+            run_id=ti.run_id if hasattr(ti, "run_id") else "",
```

Comment:
we can't get run_id easily here in Airflow 2.1 case.

Reply:
I deliberately added `has_attr` here to make it explicit conditional. getattr(...., "") is shorter but kinda blurry if you've done that deliberately. So my check flags any "gettattr(ti, 'run_id'" - no matter if it has default or not.

This is a small thing, and I can be convinced otherwise, but I think it's better to promote if hasattr in this case.

Reply:
Hmm. I convinced mysefl :).

It's better indeed, I think the default is "deliberate enough".


![image](https://user-images.githubusercontent.com/595491/159238762-39386fb0-3a5e-4f26-8e90-b6b6a59179c4.png)